apiserver/poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
单测里面构建镜像的时候报错了：
```
pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.
The command '/bin/sh -c poetry config virtualenvs.create false && poetry install' returned a non-zero code: 1
```